### ISSUE TITLE
improvements to processing times

### ISSUE THREAD

**Issue by andrewharvey** [mapillary/mapillary_tools#251]
### Basic information
* Release version: master
* System: `Linux`

I'd like to see some improvements in the processing time if possible, at the moment `mapillary_tools process_and_upload` spends quite a while running through:

```
Running user_process for 2612 images, skipping 0 images.
Running import_meta_data_process for 2612 images, skipping 0 images.
Running geotag_process for 2612 images, skipping 0 images.
Running sequence_process for 2612 images, skipping 0 images.
Running upload_params_process for 2612 images, skipping 0 images.
Running mapillary_image_description for 2612 images, skipping 0 images.
```

**Comment by jernejaMislej** [comment]
hi, yes we will work on improving the processing time, it is not using threading as upload directly, so we likely start with that. Any suggestions are welcome

**Comment by andrewharvey** [comment]
The `mapillary_image_description` stage takes maybe 10 times the time as the other stages, I'll try to run some benchmarking...

**Comment by jernejaMislej** [comment]
yes, we are noticing the same, it is due to writing  data into the image,  so we are considering ways to improve or omit that completely, you are welcome to contribute
